src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.BitsBuffer.cs

```diff
@@ -92,6 +95,7 @@ public void Reduce(ref FastReducer reducer)
             {
                 // Executes a modulo operation using an optimized reducer.
                 // Thus, no need of any switching here, happens in-line.
+
```

Comment:
As I wrote these lines of code I put some extra space below comments with a more "general character". Means: if a comment is just related to the line below, there is no blank line. If a comment is an overall note to better understand the current function, it is an extra block. Same "problem" as when to group some lines of code using new-lines or not.

As I said, they got lost and I did an revert, re-added them. I can remove them, if the change is too peculiar.